pkg/config/testdata/slice-test.yaml

```diff
@@ -0,0 +1,29 @@
+mixins:
+  - exec
+
+name: HELLO
+version: 0.1.0
+description: "An example Porter configuration"
+invocationImage: jeremyrickard/porter-hello:latest
+
+parameters:
+  - name: command 
+    type: string
+    default: "echo Hello World"
+
+install:
+  - description: "Install Hello World"
+    exec:
+      command: bash
+      arguments:
+        - -c
+        - source:  bundle.parameters.command
```

Comment:
This looks natural to me, based on how we do source elsewhere.

Ok I've said it 3 ways. I'll stop now. 😇

Reply:
Yeah, to support this form, the change i added is needed. Both this and "source:...." are valid YAML, so I think we should support both for right now.

We should probably fix this in general though..but also:

```yaml
arguments: 
   - asdadadad
   - {bundle.params.blah}
```

will probably have similar stupid YAML processing outcomes for example, that block above gets turned into:

```yaml 
args: 
  - blah
  - 
    bundle.parameters.blah: ~
```

by a validator, which is even worse. I think in that case it might require quotes to make things simpler.

Reply:
rawr 🦁 Ok I see why this is hard.

I say ship it and then reasses later.